modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/index.jsx

```diff
@@ -68,13 +64,16 @@ const SetupLiferayExperienceCloudPage = ({
 	};
 
 	const addHighPriorityContacts = (contactList) => {
-		setAddHighPriorityContactList((oldList) => {
+		setAddHighPriorityContact((oldList) => {
```

Comment:
Don't use:
1. old
2. List

In this case, we use `prev` and the name of state, in this case highPriorityContact

```suggestion
		setAddHighPriorityContact((prevHighPriorityContacts) => {
```